.github/workflows/CI-unixish.yml

```diff
@@ -446,8 +446,6 @@ jobs:
           python3 ../naming.py --var='[a-z].*' --function='[a-z].*' naming_test.c.dump
           ../../cppcheck --dump naming_test.cpp 
           python3 ../naming.py --var='[a-z].*' --function='[a-z].*' naming_test.cpp.dump
-          ../../cppcheck --dump namingng_test.c 
```

Comment:
We should at least keep a basic test to make sure it still works in standalone mode.